include/sound/sof/dai-intel.h

```diff
@@ -49,6 +49,9 @@
 /* bclk idle */
 #define SOF_DAI_INTEL_SSP_CLKCTRL_BCLK_IDLE_HIGH	BIT(5)
 
+/* DMIC max. four controllers for eight microphone channels */
```

Comment:
I thought it was 2 and 4 channels respectively? Which device supports 8  channels?

Reply:
Only Sue Creek of SOF supported platforms supports 8 channels. The platforms for notebooks are 4 channels. If Sue Creek is no more supported the macro value could be 2.

Reply:
ah, ok. that's fine then.